src/views/HomeView.vue

```diff
@@ -1,41 +1,66 @@
 <template>
   <div class="home-container">
-    <h1>Schule 4.0 - Haustiere</h1>
-    <p>Wähle ein Haustier um das Spiel zu starten:</p>
-    <select v-model="selectedPet" @change="goToPetStages">
-      <option disabled value="">Wähle ein Haustier</option>
-      <option value="dog">Hund</option>
-    </select>
+    <div class="header-box">
+      <h1>Mein erstes Haustier</h1>
+      <img src="../assets/icon_audio.svg" alt="Audio">
+    </div>
+
+    <p>Bereit, in die Welt der Haustiere einzutauchen?
+    <br>Tippe auf das Tier, das dich am meisten interessiert!</p>
+  </div>
+
+  <div class="pet-cards">
+      <PetCardComponent v-for="(pet, index) in pets" :key="index" :pet="pet" @pet-selected="handlePetSelected" />
   </div>
 </template>
 
 <script setup lang="ts">
-import { ref } from 'vue'
 import { useRouter } from 'vue-router'
+import PetCardComponent from '@/components/PetCardComponent.vue';
+import pets from '@/config/petsConfig';
 
-const selectedPet = ref('')
 const router = useRouter()
 
-const goToPetStages = () => {
-  if (selectedPet.value) {
-    router.push(`/pets/${selectedPet.value}/stages/1`)
+const handlePetSelected = (selectedPet: any) => {
+  if (selectedPet.name) {
+    router.push(`/pets/${selectedPet.englishName}/stages/1`) // only works for dog
   }
-}
+};
+
 </script>
 
 <style scoped>
 .home-container {
   margin: auto;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  padding: 20px;
+  align-items: left;
+  font-size: 30px;
+  line-height: 100%;
+
+  h1 {
+    font-size: 3rm;
+    padding: 25px 0 20px 0;
+  }
+
+  p {
+    padding: 20px 0;
+  }
+
+  .header-box {
```

Comment:
![image](https://github.com/schule4-0/pets/assets/131030998/53539c2b-22d5-44ea-9af2-b6a05366f746)
Height of the audio icon should be the same as the header (or a fixed height and vertically centered).
Audio icon color should probably also match the headers color.